src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -299,22 +282,28 @@ private void createBindPanel() {
     bindPanel.setBorder(b);
 
     JLabel projectListLabel = new JLabel("SonarQube project:");
-    projectComboBox = new ComboBoxWithWidePopup(new DefaultComboBoxModel<RemoteModule>());
-    projectComboBox.setRenderer(new ProjectComboBoxRenderer());
-    projectComboBox.setEditable(false);
-    projectComboBox.addItemListener(new ProjectItemListener());
+    projectList = new JBList<>();
+    projectList.setCellRenderer(new ProjectListRenderer());
+    projectList.addListSelectionListener(new ProjectItemListener());
+    projectList.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
+    new ListSpeedSearch(projectList, new Convertor<Object, String>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this anonymous inner class a lambda [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1604)